src/components/logPage/LogPage.jsx

```diff
@@ -14,24 +33,29 @@ const LogPage = () => {
       </section>
       <section className="logPageRight">
         <h1 className="logPageText3">Créer votre compte </h1>
-        <form>
+        <form onSubmit={(event) => {
+          post();
+          event.preventDefault();
+        }}
+        >
           <div className="logPageFormInputDiv">
             <label htmlFor="email" />
-            <input className="logPageFormInput" type="email" name="mail" id="email" placeholder="Adresse mail" required />
+            <input className="logPageFormInput" type="email" name="mail" id="email" placeholder="Adresse mail" onChange={(event) => setEmail(event.target.value)} required />
           </div>
           <div className="logPageFormInputDiv">
-            <label htmlFor="name" />
-            <input className="logPageFormInput" type="text" name="name" id="name" placeholder="Nom" required />
+            <label htmlFor="password" />
+            <input className="logPageFormInput" type="password" name="password" id="password" placeholder="Mot de passe" onChange={(event) => setPassword(event.target.value)} required />
           </div>
           <div className="logPageFormInputDiv">
             <label htmlFor="ordinal_number" />
-            <input className="logPageFormInput" type="text" name="ordinal_number" id="ordinal_number" placeholder="Numéro ordinal" required />
+            <input className="logPageFormInput" type="text" name="ordinal_number" id="ordinal_number" placeholder="Numéro ordinal" onChange={(event) => setOrdinalNumber(event.target.value)} required />
           </div>
           <p className="logPageFormText">Déjà membre ? Connectez-vous ici</p>
```

Comment:
since we are in the login page, we already are members: this message should not be here